spec/ask_export/csv_splitter_spec.rb

```diff
@@ -0,0 +1,83 @@
+require "active_support/testing/time_helpers"
+
+RSpec.describe AskExport::CsvSplitter do
+  include ActiveSupport::Testing::TimeHelpers
+
+  describe ".call" do
+    around do |example|
+      expect { example.run }.to output.to_stdout
+    end
+
+    around do |example|
+      travel_to(Time.parse("2020-05-01 11:00 UTC")) { example.run }
+    end
+
+    before do
+      allow(File).to receive(:write)
+    end
+
+    let(:live_export_path) do
+      File.join(__dir__, "../support/files/live_export.csv")
+    end
+
+    let(:draft_export_path) do
+      File.join(__dir__, "../support/files/draft_export.csv")
+    end
+
+    it "uses CsvBuilder to write files for each partner" do
+      responses = [{ id: "100000001",
```

Comment:
We're repeating this hash at least 5 times across all the specs. Can we move it into a 'factory' helper? I think that would also make this a bit easier to review; each time I see a hash, it takes a bit of thought to understand it's the same as the others.

Reply:
Sure, I've added a helper.